package.json

```diff
@@ -74,6 +74,7 @@
     "express-healthcheck": "0.1.0",
     "express-pino-logger": "4.0.0",
     "express-session": "1.17.0",
+    "fast-xml-parser": "3.16.0",
```

Comment:
This can get moved to `"devDependencies"` since we only need it for testing.

Reply:
Done in https://github.com/Seneca-CDOT/telescope/pull/644/commits/88ca58f77ff5b4e398d6e0fbf8ac1cc57774fbd3